main.tf

```diff
@@ -206,7 +206,7 @@ EOF
 }
 
 resource "aws_instance" "int_tableau_linux_staging" {
-  count                       = "${var.environment == "prod" ? "1" : "0"}"         # Allow different instance count in prod and notprod
+  count                       = "${var.environment == "prod" ? "1" : "1"}"         # Allow different instance count in prod and notprod
```

Comment:
Shouldn't we have count = 1, since it all scenarios it will be 1? (prod or not prod, so we don't need ${var.environment == "prod" ?

Reply:
We are temporarily increasing the count for testing. And will later revert the count.

Reply:
Also the comment is wrong. "Allow different instance count in prod and notprod".